src/ui/map.js

```diff
@@ -122,6 +124,7 @@ const defaultOptions = {
  * @param {number} [options.bearing=0] The initial bearing (rotation) of the map, measured in degrees counter-clockwise from north. If `bearing` is not specified in the constructor options, Mapbox GL JS will look for it in the map's style object. If it is not specified in the style, either, it will default to `0`.
  * @param {number} [options.pitch=0] The initial pitch (tilt) of the map, measured in degrees away from the plane of the screen (0-60). If `pitch` is not specified in the constructor options, Mapbox GL JS will look for it in the map's style object. If it is not specified in the style, either, it will default to `0`.
  * @param {boolean} [options.renderWorldCopies=true]  If `true`, multiple copies of the world will be rendered, when zoomed out.
+  * @param {number} [options.tileCacheSize=null]  The number of tiles stored in the tile cache. If omitted, the cache will be dynamically sized based on the current viewport.
```

Comment:
In light of the comment on the logic in `SourceCache`, would it make sense to rename this to `maxTileCacheSize`?  (Sorry, I know you already renamed once! 😬 )